presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt

```diff
@@ -199,10 +199,10 @@ fun FeedBackScreen(
                                     modifier = Modifier
                                         .height(40.dp)
                                         .width(40.dp),
-                                    painter = painterResource(id = R.drawable.smiling),
+                                    painter = painterResource(id = R.drawable.ic_feedback_smiling_face),
                                     contentDescription = stringResource(id = R.string.sign_in_label)
                                 )
-                                Text(text = "Bad")
+                                Text(text = "Great")
```

Comment:
can we extract this to string resource?

Reply:
Yes. I'll fix that.

Reply:
@chepsi Done. Have a look a the changes.